lib/phonelib/phone.rb

```diff
@@ -78,6 +79,11 @@ def country
       end
     end
 
+    # Returns the country code from the original phone number.
+    def country_code
+      @data.values.first[:country_code]
```

Comment:
@mastahyeti Can you base this result on `country` method and please include a check if there is no valid country with test for it, thanks